app/org/sagebionetworks/bridge/services/SurveyService.java

```diff
@@ -172,15 +180,11 @@ public Survey versionSurvey(GuidCreatedOnVersionHolder keys) {
      * survey version that could have been sent to users will remain in the API so you can look at its 
      * schema, etc. This is how study developers should delete surveys. 
      */
-    public void deleteSurvey(GuidCreatedOnVersionHolder keys) {
-        checkArgument(StringUtils.isNotBlank(keys.getGuid()), "Survey GUID cannot be null/blank");
-        checkArgument(keys.getCreatedOn() != 0L, "Survey createdOn timestamp cannot be 0");
-
+    public void deleteSurvey(StudyIdentifier studyIdentifier, GuidCreatedOnVersionHolder keys) {
         Survey existing = surveyDao.getSurvey(keys, true);
-        if (existing.isDeleted()) {
+        if (existing == null || existing.isDeleted() || !isInStudy(studyIdentifier, existing)) {
             throw new EntityNotFoundException(Survey.class);
         }
```

Comment:
Currently none of the integration tests try and do this so it's not causing anything to fail. I would prefer to implement a cross-study admin (see PR) and change the integration tests to use that, rather than fully implement a role-based exception here.